tests/integration/conftest.py

```diff
@@ -4,11 +4,17 @@
 import subprocess
 from pathlib import Path
 
+import jubilant
 import yaml
 from pytest import fixture
 
 from . import TRAEFIK
-from .juju import Juju
+
+
+@fixture
```

Comment:
I believe you're looking for a module-scoped fixture. That's why the tests in `test_path_ingress_traefik.py` are failing, I think -- because the subsequent tests are expecting the previous tests to have set things up (deployed) already, but with this fixture the way it is, you get a new model every test.

Reply:
Yeh I realised just as you were posting! Will adjust.